interfaces/apparmor/backend.go

```diff
@@ -704,16 +704,20 @@ func addContent(securityTag string, snapInfo *snap.Info, cmdName string, opts in
 	}
 	policy = templatePattern.ReplaceAllStringFunc(policy, func(placeholder string) string {
 		switch placeholder {
+		case "###INCLUDEIFEXISTSSNAPTUNING###":
+			features, _ := parserFeatures()
+			if strutil.ListContains(features, "include-if-exists") {
+				return `#include if exists "/var/lib/snapd/apparmor/snap-tuning"`
```

Comment:
I'm super-excited that apparmor3 is now being built into the snapd snap like this. Nice! This `snap-tuning` seems like an unrelated change. How is this meant to be used? Is it documented somewhere?

Reply:
It's meant to be used to drop in additional apparmor rules like we recommended folks who have non-standard $HOME setups to do, since the recommendation in i.e. https://forum.snapcraft.io/t/how-can-i-use-snap-when-i-dont-use-home-user/3352 doesn't work anymore with apparmor bundled inside the snapd snap as the options we pass to apparmor_parser specifically only include dirs etc from the snapd snap and not from the host. So now the recommendation will be to put such tunables inside this specific file/directory instead.

I think we do intend to document this, but there were some unresolved questions about the support level of that file...